ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -0,0 +1,586 @@
+import React, { useContext, useEffect, useMemo, useState } from 'react';
```

Comment:
This converted the old `network-list-menu.js` to typescript and simplified it + added features.  Although git interpreted it as a delete + add